project/Build.scala

```diff
@@ -79,7 +83,9 @@ object DottyBuild extends Build {
 
       tuning ::: agentOptions ::: travis_build ::: fullpath
     }
-  )
+  ) ++ (if (isTravisBuild) Seq() else addCommandAlias("test", "partest"))
```

Comment:
Can we leave the chance to have both? Ie to be able to either execute tests with partest or with JUnit from sbt?